### PR TITLE
Phoenix/all locations

### PR DESCRIPTION
**Pull request by phoenixAja**
added new workflow files for new regions

### CODE REVIEW

File: bin/launch-build [hunk 1]
```diff
@@ -11,5 +11,5 @@ nextstrain build --aws-batch --detach \
     "$@" \
     --profile profiles/nextstrain \
     --cores "$NEXTSTRAIN_AWS_BATCH_CPUS" \
-    --resources mem_mb="$NEXTSTRAIN_AWS_BATCH_MEMORY" \
-    --set-threads tree=64
+    --resources mem_mb="$NEXTSTRAIN_AWS_BATCH_MEMORY" #\
+    # --set-threads tree=64
```
Comment (ttung) [inline bin/launch-build:15]: we can leave this.  even if we set tree building threads=64, it'll be capped by the number of cores given by `--cores`

File: .github/workflows/ingest_california.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
     - name: ingest
       run: |
         PATH="$HOME/.local/bin:$PATH"
-        pipenv run ./bin/ingest
+        pipenv run ./bin/ingest division California
```
[the remark this answers is not in the document]
Comment (phoenixAja) [reply .github/workflows/ingest_california.yml:29]: the frequency i set as weekly (midnight on saturday)

File: bin/ingest [hunk 3]
```diff
@@ -45,23 +55,29 @@ set -x
 if [[ "$branch" == master ]]; then
     ./bin/notify-on-gisaid-change data/gisaid.ndjson "$S3_SRC/gisaid.ndjson.gz"
 fi
-./bin/upload-to-s3 --quiet data/gisaid.ndjson "$S3_DST/gisaid.ndjson.gz"
+
+# only copy gisaid data if region is California to avoid storing duplicate files
+if [ "California" == "$region" ]; then
+  ./bin/upload-to-s3 --quiet data/gisaid.ndjson "$S3_DST/gisaid.ndjson.gz"
```
[the remark this answers is not in the document]
Comment (phoenixAja) [reply bin/ingest:64]: I'll change these, my indentation was set at 2 spaces

File: bin/ingest [hunk 4]
```diff
@@ -45,23 +55,29 @@ set -x
...
-    ./bin/notify-on-additional-info-change data/additional_info.tsv "$S3_SRC/additional_info.tsv.gz"
 
-    ./bin/rebuild-staging data/metadata.tsv "$S3_SRC/metadata.tsv.gz" \
-      data/sequences.fasta "$S3_SRC/sequences.fasta.gz"
+    ./bin/notify-on-metadata-change data/metadata.tsv "$S3_SRC/${region}/metadata.tsv.gz"
+    ./bin/notify-on-additional-info-change data/additional_info.tsv "$S3_SRC/${region}/additional_info.tsv.gz"
+
+    ./bin/rebuild-staging data/metadata.tsv "$S3_SRC/${region}/metadata.tsv.gz" \
+      data/sequences.fasta "$S3_SRC/${region}/sequences.fasta.gz" $specifier $region
+
```
[the remark this answers is not in the document]
Comment (phoenixAja) [reply bin/ingest]: rebuild-staging? yeah it should, the only thing that shouldn't happen is uploading the gisaid data and slack notifications

File: .github/workflows/ingest_california.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (phoenixAja) [reply .github/workflows/ingest_california.yml:29]: frequency i put as weekly, midnight on Saturday

File: bin/ingest [hunk 5]
```diff
@@ -45,23 +55,29 @@ set -x
...
 fi
 
-./bin/upload-to-s3 ${silent:+--quiet} data/metadata.tsv "$S3_DST/metadata.tsv.gz"
-./bin/upload-to-s3 ${silent:+--quiet} data/additional_info.tsv "$S3_DST/additional_info.tsv.gz"
-./bin/upload-to-s3 ${silent:+--quiet} data/sequences.fasta "$S3_DST/sequences.fasta.gz"
+./bin/upload-to-s3 ${silent:+--quiet} data/metadata.tsv "$S3_DST/${region}/metadata.tsv.gz"
+./bin/upload-to-s3 ${silent:+--quiet} data/additional_info.tsv "$S3_DST/${region}/additional_info.tsv.gz"
+./bin/upload-to-s3 ${silent:+--quiet} data/sequences.fasta "$S3_DST/${region}/sequences.fasta.gz"
```
[the remark this answers is not in the document]
Comment (phoenixAja) [reply bin/ingest:88]: i thought different regions would have different data for these files, if it's all the same i can remove this, i was trying to organize inputs and outputs for each region

[the remark this answers is not in the document]
Comment (phoenixAja) [reply bin/ingest:88]: yeah we don't need this by region, i'll take out

**Comment by phoenixAja** [comment]
[Minnesota results zipfile on s3](https://s3.console.aws.amazon.com/s3/object/covid19-nextstrain-all-locations-batch/ef12c7e4-ced8-4da6-9494-9ca6865d7aff.zip?region=us-west-2&tab=overview)

when viewing results with `nextstrain view ef12c7e4-ced8-4da6-9494-9ca6865d7aff/auspice` it looks like the reads weren't filtered for Minnesota. @ttung were you getting proper filtering before with `division!=California`? Looks like the build is all north america.

Screenshot 2020-06-04 13 42 22

**Comment by ttung** [comment]
I made a few changes and I'll land it now.

**Comment by ttung** [comment]
See https://github.com/czbiohub/ncov-ingest/pull/1/commits/3dea3c7819e26443b32a4e0e4c49afcf3c53cb25

**Pull request merged**